platformshconfig/config.py

```diff
@@ -362,8 +362,10 @@ def get_route(self, route_id):
     def application(self):
         """Returns the application definition dict.
 
-        This is, approximately, the .platform.app.yaml file as a nested dict. However, it also has other information
-        added by Platform.sh as part of the build and deploy process.
+           @deprecated
```

Comment:
To be fair, I don't know if @deprecated is as common a marker in Python as it is in PHP and JS and other JavaDoc-inpsired languages.  I'm good with using it but there may be some other Python specific convention.